src/frontend/src/tests/lib/components/ui/InputCurrency.spec.ts

```diff
@@ -0,0 +1,54 @@
+import InputCurrency from '$lib/components/ui/InputCurrency.svelte';
+import { assertNonNullish } from '@dfinity/utils';
+import { fireEvent, render } from '@testing-library/svelte';
+
+describe('ConvertAmountExchange', () => {
```

Comment:
I think here is InputCurrency, right?

Reply:
Totally right, but since we will most likely proceed with the Solution 2, this PR won't be merged.